### PR TITLE
ci(lintcommit): use nvim -l

### PR DESCRIPTION
**Pull request by lewis6991**

### CODE REVIEW

File: .github/workflows/lintcommit.yml [hunk 1]
```diff
@@ -18,4 +18,4 @@ jobs:
       - uses: rhysd/action-setup-vim@v1
         with:
           neovim: true
-      - run: nvim --clean -es +"lua require('scripts.lintcommit').main({trace=false})"
+      - run: nvim --clean -l scripts/lintcommit.lua main -trace
```
Comment (clason) [inline .github/workflows/lintcommit.yml:21]: If we change this to have `trace` enabled by default, we might as well remove the option and just be verbose always.

Comment (lewis6991) [reply .github/workflows/lintcommit.yml:21]: It's not enabled `-trace` disables, `+trace` enables. We can use a different syntax if you prefer?

Comment (clason) [reply .github/workflows/lintcommit.yml:21]: `--trace` and  `--no-trace` is more standard in (shell) scripts?

Comment (lewis6991) [reply .github/workflows/lintcommit.yml:21]: Sure

Comment (clason) [reply .github/workflows/lintcommit.yml:21]: But no strong opinions either way, I just didn't understand the syntax.

**Review by dundargoc — DISMISSED** [review]
`make lint` fails

**Comment by dundargoc** [comment]
Also fix comment at top of file

**Review by dundargoc — CHANGES_REQUESTED** [review]
`make lint` fails

**Comment by lewis6991** [comment]
Does it?

**Comment by dundargoc** [comment]
For me it does. Does it not for you?

**Comment by lewis6991** [comment]
CI passed. Does that not do `make lint`?

**Comment by lewis6991** [comment]
`make lint` still fails, but only because the cmake target runs on all the commit messages.

**Pull request merged**